packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -307,6 +331,15 @@ export class InsuredBridgeL1Client {
       this.instantRelays[l1Token] = {};
     }
 
+    // Fetch and store all rate model updated events, which will be used to fetch the rate model for a specific deposit
+    // quote timestamp.
+    // Note: there is no need to sort these events chronologically, as they will be sorted upon calling
+    // `rateModel.updateRateModelEventDictionary`
+    this.updatedRateModelEventsForToken = this.updatedRateModelEventsForToken.concat(
+      await this._getAllRateModelEvents(blockSearchConfig)
```

Comment:
can this async call be bundled with some of the others? perhaps placed in the promise.all block below or joined with the call to fetch the OO liveness?

Reply:
done